### PR TITLE
[EOC-196] editing list name and description on mobile additional save button

### PR DESCRIPTION
**Pull request by alemikolo**
Screenshot 2019-06-05_13-28-09-348
Screenshot 2019-06-05_13-28-26-894
Screenshot 2019-06-05_13-28-59-083

### CODE REVIEW

File: client/src/modules/cohort/components/CohortHeader.jsx [hunk 1]
```diff
@@ -159,6 +170,17 @@ class CohortHeader extends PureComponent {
...
+  handleTipVisibility = () => {
+    const { nameInputValue } = this.state;
+    const isItemNameEmpty = !_trim(nameInputValue);
+
+    if (isItemNameEmpty) {
+      this.setState({ isTipVisible: true });
+    }
+  };
+
+  hideTipAfterTimeout = () => this.setState({ isTipVisible: false });
```
Comment (adamklepacz) [inline client/src/modules/cohort/components/CohortHeader.jsx]: can't see any timeout here?
change name to just hideTip

File: client/src/modules/list/components/ListHeader.jsx [hunk 2]
```diff
@@ -179,6 +190,17 @@ class ListHeader extends PureComponent {
...
+  handleTipVisibility = () => {
+    const { nameInputValue } = this.state;
+    const isItemNameEmpty = !_trim(nameInputValue);
+
+    if (isItemNameEmpty) {
+      this.setState({ isTipVisible: true });
+    }
+  };
+
+  hideTipAfterTimeout = () => this.setState({ isTipVisible: false });
```
Comment (adamklepacz) [inline client/src/modules/list/components/ListHeader.jsx]: no timeout here,
change name

File: client/src/modules/list/components/ListHeader.jsx [hunk 3]
```diff
@@ -228,19 +250,27 @@ class ListHeader extends PureComponent {
...
 
     return isNameInputVisible ? (
-      <NameInput
-        disabled={pendingForName}
-        name={nameInputValue}
-        onClick={isOwner ? this.handleClick : null}
-        onKeyPress={this.handleKeyPress}
-        onNameChange={this.handleNameChange}
-      />
+      <div>
```
Comment (adamklepacz) [inline client/src/modules/list/components/ListHeader.jsx]: maybe use <fragment/> here

File: client/src/modules/list/components/ListHeader.jsx (hunk 2 above)
Comment (alemikolo) [reply client/src/modules/list/components/ListHeader.jsx]: the hint hides when the user enters something

File: client/src/modules/list/components/ListHeader.jsx (hunk 3 above)
Comment (alemikolo) [reply client/src/modules/list/components/ListHeader.jsx]: I can't use <Fragment> here. The container of this div is displayed flex with flex-direction: row, so without this div (and with Fragment), the hint appears next to the input field (between the input field and functionality of changing list's type) but not below it.

**Pull request merged**